request_list_with_add_ticket_sample_app/build.gradle

```diff
@@ -0,0 +1,33 @@
+apply plugin: 'com.android.application'
+
+android {
+    compileSdkVersion 23
```

Comment:
I think we should extract all of those constants (compileSdk, support, sdk, ...), to make sure all sample apps using the same version and to keep maintenance effort low.